packages/nc-gui-v2/composables/useColumnCreateStore.ts

```diff
@@ -198,7 +198,8 @@ const [useProvideColumnCreateStore, useColumnCreateStore] = createInjectionState
       }
       onSuccess()
     } catch (e: any) {
-      toast.error(await extractSdkResponseErrorMsg(e))
+      const error = await extractSdkResponseErrorMsg(e)
+      if (error) toast.error(await extractSdkResponseErrorMsg(e))
```

Comment:
`await extractSdkResponseErrorMsg(e)` must return something. I think L201 is not necessary and you can just use `toast.error(await extractSdkResponseErrorMsg(e))`

Reply:
@wingkwong It is. In form validation error it raises an validation error which causes empty toastr to come up. Doing this will make only error handed by Noco sdk to come up.

Reply:
Maybe better handle it in `extractSdkResponseErrorMsg` (Line 2)

Reply:
@wingkwong It will add extra complications to a function which is used in a lot of places. I think handling this way is better?